services/motion/client_test.go

```diff
@@ -368,11 +368,9 @@ func TestClient(t *testing.T) {
 		})
 
 		t.Run("otherwise returns a slice of PlanStautsWithID", func(t *testing.T) {
-			planID, err := uuid.NewUUID()
-			test.That(t, err, test.ShouldBeNil)
+			planID := uuid.New()
```

Comment:
I realized that it was unnecessary to use `uuid.NewUUID` which can return an error & that everywhere else in the RDK `uuid.New` is used. Swapping these function out removed a bunch of error cases from helper functions which didn't need to exist.